### PR TITLE
Added back some commands and fixed a few bugs

### PR DESCRIPTION
**Pull request by oof2win2**
# Changes
- Added `?rollback` command back
- Added `?statss` command back
- Added `?shellcmd` command to run shell commands

# Bugfixes
- Fixed a bug where the bot would constantly be sending research (and made that sending part actually readable)
- Fixed some bugs when installing on production

### CODE REVIEW

File: commands/Administration/rollback.js [hunk 1]
```diff
@@ -26,8 +25,55 @@ class Linkme extends Command {
 	}
 
 	async run(message, args) {
+		const serverid = message.mentions.channels.first()?.id || await this.client.channels.fetch(args[0]).then(c => c?.id)
+		if (!serverid) return message.channel.send(`\`${args[0]}\` is an invalid server!`)
 
+		const server = servers.find(server => server.discordid === serverid)
+		if (!server) return message.channel.send(`\`${args[0]}\` is an invalid server!`)
```
Comment (DistroByte) [inline commands/Administration/rollback.js:32]: this line and line 29 are the exact same. remove one of them if possible

File: commands/Factorio/statss.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	constructor(client) {
+		super(client, {
+			name: "statss",
+			description: "Get statistics of a Factorio server",
+			usage: "[channel]",
+			examples: ["{{p}}statss #awf-regular", "{{p}}statss 724696348871622818"],
+			dirname: __dirname,
+			enabled: true,
+			guildOnly: false,
+			aliases: ["fstatsserver", "factorioserverstatistics"],
```
Comment (DistroByte) [inline commands/Factorio/statss.js:16]: another clear alias would be `serverstats`

**Review by DistroByte — COMMENTED** [review]
just fix the few bits I pointed out!

**Comment by oof2win2** [comment]
Fixed!

**Review by DistroByte — APPROVED** [review]

**Pull request merged**